client/src/glossary/glossary.js

```diff
@@ -84,46 +113,60 @@ const Glossary_ = ({ active_key, items_by_letter }) => (
       </div>
       <div className="glossary-items">
         <dl>
-          {_.map(items_by_letter, ({ letter, items}) => _.map(items, (item,ix) => 
-            <Fragment key={ix}>
-              <dt
-                className="glossary-dt"
-                id={
-                ix === 0 ?
-                `__${letter}` :
-                null 
-                }
-                tabIndex={0}
-              >
-                <span
-                  id={item.id}
-                  tabIndex={-1}
-                >
-                  {item.title}
-                </span>
-              </dt>
-              <dd>
-                <div 
-                  dangerouslySetInnerHTML={{__html: item.definition}}
-                />
-                <div>
-                  <p>{text_maker("glossary_translation")} {item.translation}</p>
-                </div>
-                
-                <div className='glossary-top-link-container'>
-                  <a className="glossary-top-link"
-                    href="#"
-                    tabIndex='0'
-                    onClick={evt => {
-                      evt.preventDefault();
-                      document.body.scrollTop = document.documentElement.scrollTop = 0;
-                      document.querySelector('#glossary_search > div > div > input').focus();
-                    }}
-                  >{text_maker("back_to_top")}</a>
-                </div>
-              </dd>
-              
-            </Fragment>))}
+          {_.map(
+            items_by_letter, 
+            ({letter, items}) => _.map(
+              items, 
+              (item, ix) => (
+                <Fragment key={ix}>
+                  <dt
+                    className="glossary-dt"
+                    id={
+                    ix === 0 ?
+                    `__${letter}` :
+                    null 
+                    }
```

Comment:
Couldn't this be on one line?

Reply:
Ugh, yeah, the indenting is total trash too. I didn't write that, just moved it, will fix though.